packages/vapor/scss/redesign/variables.scss

```
@@ -192,7 +192,8 @@
 
     // Badge
     --badge-background-color: var(--orange);
-    --badge-text-color: var(--pure-white);
+    --badge-font-size: 0.92em;
```

shouldn't it be the following?
```suggestion
    --badge-font-size: #{$badge-font-size};
```

---

It becomes an undefined variable. I think this is the only area where `--badge-font-size` is determined here and that's where the `#{$badge-font-size}` is evaluated for the redesign:

https://github.com/coveo/react-vapor/blob/master/packages/vapor/scss/redesign/fonts.scss#L33